src/main/java/com/github/wovnio/wovnjava/Api.java

```diff
@@ -23,14 +24,17 @@
     private final int READ_BUFFER_SIZE = 8196;
     private final Settings settings;
     private final Headers headers;
+    private final ResponseHeaders responseHeaders;
     private final String responseEncoding = "UTF-8"; // always response is UTF8
 
-    Api(Settings settings, Headers headers) {
+    Api(Settings settings, Headers headers, ResponseHeaders responseHeaders) {
         this.settings = settings;
         this.headers = headers;
+        this.responseHeaders = responseHeaders;
     }
 
     String translate(String lang, String html) throws ApiException {
+        this.responseHeaders.setApi("Requested");
```

Comment:
if you feel like it, i'd rename the setters to `setApiStatus`, `setApiStatusCode`